QGL/ChannelLibrary.py

```diff
@@ -105,15 +109,25 @@ def load_from_library(self):
                     # connect objects labeled by strings
                     for chan in tmpLib.channelDict.values():
                         if hasattr(chan, 'physChan'):
-                            chan.physChan = tmpLib[chan.physChan] if chan.physChan in tmpLib.channelDict else None
+                            chan.physChan = tmpLib[
+                                chan.
+                                physChan] if chan.physChan in tmpLib.channelDict else None
                         if hasattr(chan, 'gateChan'):
-                            chan.gateChan = tmpLib[chan.gateChan] if chan.gateChan in tmpLib.channelDict else None
+                            chan.gateChan = tmpLib[
+                                chan.
+                                gateChan] if chan.gateChan in tmpLib.channelDict else None
                         if hasattr(chan, 'trigChan'):
-                            chan.trigChan = tmpLib[chan.trigChan] if chan.trigChan in tmpLib.channelDict else None
+                            chan.trigChan = tmpLib[
+                                chan.
+                                trigChan] if chan.trigChan in tmpLib.channelDict else None
                         if hasattr(chan, 'source'):
-                            chan.source = tmpLib[chan.source] if chan.source in tmpLib.channelDict else None
+                            chan.source = tmpLib[
+                                chan.
+                                source] if chan.source in tmpLib.channelDict else None
                         if hasattr(chan, 'target'):
-                            chan.target = tmpLib[chan.target] if chan.target in tmpLib.channelDict else None
+                            chan.target = tmpLib[
+                                chan.
+                                target] if chan.target in tmpLib.channelDict else None
```

Comment:
I guess this whole series would be more cleanly spelled as, for example,  `tmpLib.get(chan.target)` which returns `None` if the key does not exist.